neptune/driver/healthcheck.go

```diff
@@ -12,3 +19,20 @@ func (n *NeptuneDriver) Healthcheck() (s string, err error) {
 	}
 	return serviceName, nil
 }
+
+// Checker hecks health of Neo4j and updates the provided CheckState accordingly
```

Comment:
Comment should contain `Neptune` instead of `Neo4j`:
`Checker checks the health of Neptune and updates the provided CheckState accordingly`